client/core/trade.go

```diff
@@ -1584,6 +1584,26 @@ func (c *Core) swapMatchGroup(t *trackedTrade, matches []*matchTracker, errs *er
 		errs.add("not broadcasting swap while DEX %s connection is down (could be revoked)", t.dc.acct.host)
 		return
 	}
+
+	// Use a higher swap fee rate if a local estimate is higher than the
+	// prescribed rate, but not higher than the funded (max) rate.
+	if highestFeeRate < t.metaData.MaxFeeRate {
+		var freshRate uint64
+		if r, ok := t.wallets.fromWallet.feeRater(); ok {
+			freshRate, _ = r.FeeRate()
```

Comment:
Felt like extra noise, at least until we resolve https://github.com/decred/dcrdex/issues/1484

Reply:
How is that related?

Reply:
It makes an error from dcr spv

Reply:
Ah ok, I see.

Reply:
https://github.com/decred/dcrdex/pull/1430#discussion_r797969311

Reply:
I think we should consider changing the signature of FeeRate so that it has no error return.  Callers should simply handle a zero return, and the implementation can log internally is there's actually something wrong

Reply:
Sounds reasonable to me.

Reply:
Created issue https://github.com/decred/dcrdex/issues/1516